client/src/pages/dashboard/components/Nhl/DetailPage/index.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { connect } from "react-redux";
+import { Button, Row, Card } from "reactstrap";
+import { string } from "prop-types";
+import axios from "axios";
+import styled from "styled-components";
+
+import Schedule from "../Schedule";
+import TeamStats from "../TeamStats";
+import Roster from "../Roster";
+import DetailPageModal from "./DetailPageModal";
+
+const CircleButton = styled(Button)`
+  border-radius: 1000px;
+`;
+
+const StyleCard = styled(Card)`
+  display: flex;
+  padding: 20px;
+  margin-bottom: 20px;
+`;
+
+const DivFlex = styled.div`
+  display: flex;
+  align-items: center;
+  padding: 20px 0;
+`;
+
+const DetailPage = ({ teamId }) => {
+  const [teamName, setTeamName] = useState("");
+  const [roster, setRoster] = useState([]);
+  const [modal, setModal] = useState(false);
+  const [alertData, setAlertData] = useState("");
+  useEffect(() => {
+    axios
+      .get(
+        `https://statsapi.web.nhl.com/api/v1/teams/${teamId}?expand=team.roster`
+      )
+      .then((response) => {
+        setRoster(response.data.teams[0].roster.roster);
```

Comment:
Please add checking for `response.data.teams[0]`

Reply:
https://github.com/dimapanasiuk/mern/commit/4c324997f2d564886932c4c5b210ebc25b5520c4